kaos/algorithm/cubic_equation_solver.py

```diff
@@ -0,0 +1,108 @@
+# CUBIC ROOT SOLVER
+
+# Date Created   :    24.05.2017
+# Created by     :    Shril Kumar [(shril.iitdhn@gmail.com),(github.com/shril)] &
+#                     Devojoyti Halder [(devjyoti.itachi@gmail.com),(github.com/devojoyti)]
+
+# Project        :    Classified
+# Use Case       :    Instead of using standard numpy.roots() method for finding roots,
+#                     we have implemented our own algorithm which is ~10x faster than
+#                     in-built method.
+
+# Algorithm Link :    www.1728.org/cubic2.htm
+
+# This script (Cubic Equation Solver) is an independent program for computation of cubic equations.
+# This script, however, has no relation with original project code or calculations.
+# It is to be also made clear that no knowledge of it's original project is included or used to
+# device this script. This script is complete freeware developed by above signed users, and may
+# further be used or modified for commercial or non - commercial purpose.
+
+
+# Libraries imported for fast mathematical computations.
+import math
+import numpy as np
+
+# Main Function takes in the coefficient of the Cubic Polynomial
+# as parameters and it returns the roots in form of numpy array.
+# Polynomial Structure -> ax^3 + bx^2 + cx + d = 0
+# IMPORTANT NOTE: Function has been modified to ignore imaginary numbers and repeated roots
+
+
+def solve(a, b, c, d):
+    if (a == 0 and b == 0):                     # Case for handling Liner Equation
+        return np.array([(-d * 1.0) / c])                 # Returning linear root as numpy array.
+
+    elif (a == 0):                              # Case for handling Quadratic Equations
+
```

Comment:
I know this is an open source file but we should make it conform to standard, remove blank line